goprocessor/build.sh

```diff
@@ -0,0 +1,3 @@
+#!/usr/bin/env bash
+
+go build src/sawtooth_siimplewallet/simple_wallet.go
```

Comment:
please keep "simple_wallet" as a single word "simplewallet" to maintain consistency with TPs in other languages.